org.gnome.Photos.json

```diff
@@ -252,55 +254,51 @@
                              "-Denable-tmdb=no",
                              "-Denable-tracker=no",
                              "-Denable-vimeo=no",
-                             "-Denable-youtube=no" ],
+                             "-Denable-youtube=no",
+                             "--wrap-mode=nofallback" ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/grilo-plugins/0.3/grilo-plugins-0.3.11.tar.xz",
-                    "sha256": "dde2e605b1994341c6bf012493e056b406b08571834dea3b3c671d5b8b1dcd73"
+                    "url": "https://download.gnome.org/sources/grilo-plugins/0.3/grilo-plugins-0.3.12.tar.xz",
+                    "sha256": "c6b6df086a164d65c206d70139ce80591f8feca3545612e45b823fb4fe4b2577"
                 }
             ]
         },
         {
             "name": "tracker",
             "buildsystem": "meson",
-            "cleanup": [ "/bin", "/etc", "/lib/girepository-1.0", "/libexec", "/share/dbus-1", "/share/gir-1.0" ],
+            "cleanup": [ "/bin", "/etc", "/lib/girepository-1.0", "/share/dbus-1", "/share/gir-1.0" ],
             "config-opts": [ "-Dbash_completion=no", "-Ddocs=false", "-Dsystemd_user_services=no", "-Dman=false" ],
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/tracker/2.3/tracker-2.3.4.tar.xz",
-                    "sha256": "577952244ab977c78b0b88e2f63c4197eaba16e4d66bff692b7f58993e06516d"
+                    "url": "https://download.gnome.org/sources/tracker/2.3/tracker-2.3.6.tar.xz",
```

Comment:
Photos 3.38 uses tracker3 iirc.

Reply:
It is a work in progress https://gitlab.gnome.org/GNOME/gnome-photos/-/merge_requests/135.